exporters/webhook/models/pelorus_webhook.py

```diff
@@ -44,17 +45,14 @@ class PelorusDeliveryHeaders(BaseModel):
     @validator("x_hub_signature_256", pre=True, always=True)
     def validate_x_hub_signature_256(cls, value):
         if value is not None:
-            try:
-                algorithm, signature = value.split("=", 1)
-                if algorithm != "sha256":
-                    raise ValueError("Signature should use sha256 algorithm")
-                if not signature or len(signature) != 64:
-                    raise ValueError(
-                        "Signature should be in format 'sha256=' followed by 64 characters"
-                    )
-                int(signature, 16)
-            except (TypeError, ValueError):
-                raise
+            algorithm, signature = value.split("=", 1)
+            if algorithm != "sha256":
+                raise ValueError("Signature should use sha256 algorithm")
+            if not signature or len(signature) != 64:
+                raise ValueError(
+                    "Signature should be in format 'sha256=' followed by 64 characters"
+                )
+            int(signature, 16)
```

Comment:
this line does anything?

Reply:
Yes it ensures the string is proper hex.